timeline-chart/src/layer/time-graph-chart-cursors.ts

```diff
@@ -179,11 +169,26 @@ export class TimeGraphChartCursors extends TimeGraphChartLayer {
 
     centerCursor() {
         if (this.unitController.selectionRange) {
-            const cursorPosition = this.unitController.selectionRange.start;
+            const cursorPosition = this.unitController.selectionRange.end;
             const halfViewRangeLength = this.unitController.viewRangeLength / 2;
+            let startViewRange = cursorPosition - halfViewRangeLength;
+            let endViewRange = cursorPosition + halfViewRangeLength;
+
+            let delta = 0;
```

Comment:
You don't need the delta variable here, you can just do the operations directly in each of the blocks

Reply:
updated. Thanks.